app/dashboard/views.py

```diff
@@ -3211,12 +3421,6 @@ def connect_google():
 @login_required
 @require_POST
 def request_verify_google(request, handle):
-    is_logged_in_user = request.user.is_authenticated and request.user.username.lower() == handle.lower()
```

Comment:
..

Reply:
@login_required would take care of this right ?